packages/base/src/CustomElementsScopeUtils.ts

```diff
@@ -105,11 +107,24 @@ const getEffectiveScopingSuffixForTag = (tag: string) => {
 	}
 };
 
+/**
+ * @public
+ * Used for getting a scoped name for a CSS variable using the same transformation used in the build
+ * @name the name of the css variable as written in the code
+ * @returns a variable name with the current version inserted as available at runtime
+ */
+const getScopedVarName = (name: string) => {
+	const versionStr = `v${VersionInfo.version.replaceAll(".", "-")}`;
+	const expr = /(--_?ui5)([^,:)\s]+)/g;
+	return name.replaceAll(expr, `$1-${versionStr}$2`);
+};
+
 export {
 	setCustomElementsScopingSuffix,
 	getCustomElementsScopingSuffix,
 	setCustomElementsScopingRules,
 	getCustomElementsScopingRules,
 	shouldScopeCustomElement,
 	getEffectiveScopingSuffixForTag,
+	getScopedVarName,
```

Comment:
Maybe re-export in the customer-facing module: `packages/base/src/CustomElementsScope.ts` at some point for app development (for some rare cases where apps use our vars), but for now I'm ok to keep it internal

Reply:
all other methods in this module are marked public and that made me put it here, I didn't notice the reexport. This means at the very least we are missing JS docs for marking these as private and the reexports as public